services/web/server/src/simcore_service_webserver/clusters/plugin.py

```diff
@@ -9,29 +9,22 @@
 import logging
 
 from aiohttp import web
-from servicelib.aiohttp.application_setup import (
-    ModuleCategory,
-    SkipModuleSetup,
-    app_module_setup,
-)
+from servicelib.aiohttp.application_setup import ModuleCategory, app_module_setup
 
 from .._constants import APP_SETTINGS_KEY
 from . import handlers
 
-log = logging.getLogger(__file__)
+log = logging.getLogger(__name__)
 
 
 @app_module_setup(
     "simcore_service_webserver.clusters",
     ModuleCategory.ADDON,
-    depends=["simcore_service_webserver.groups"],
+    settings_name="WEBSERVER_CLUSTERS",
```

Comment:
question: why did you remove the dependency to the groups?

Reply:
my mistake. 
The dependency mechanism will be also improved. For the moment is just used as an "assertion" mechanism at startup